### PR TITLE
v2.12.2

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/build-sysroot.sh [hunk 1]
```diff
@@ -2,22 +2,5 @@
...
-mv lib64/* lib
-rm -rf lib64
-
-mv usr/lib64/* usr/lib/
-rm -rf usr/lib64
-
-ln -s $PWD/lib $PWD/lib64
-ln -s $PWD/usr/lib $PWD/usr/lib64
-
+cp -Rf ${SRC_DIR}/x86_64-conda_cos6-linux-gnu/sysroot/* .
```
Comment (isuruf) [inline recipe/build-sysroot.sh:5]: This has way more stuff than just the sysroot.

Comment (beckermr) [reply recipe/build-sysroot.sh:5]: Happy to mark this broken and try again. I was trying to test it live.

Comment (beckermr) [reply recipe/build-sysroot.sh:5]: I added the broken label and will make another PR. What do we need to include and/or exclude here?

Comment (isuruf) [reply recipe/build-sysroot.sh:5]: Can you compare with the CDT builds? There are stuff like `libgcc-ng`, `libgomp.spec`, etc.

Comment (beckermr) [reply recipe/build-sysroot.sh:5]: Yes. I will compare and exclude what is not needed.

Comment (isuruf) [reply recipe/build-sysroot.sh:5]: Can we go with https://github.com/conda-forge/ctng-compilers-feedstock/pull/21 if we are splitting the sysroot like this?

Comment (beckermr) [reply recipe/build-sysroot.sh:5]: Yup. Let's decide amongst the options and then clean up the PR we want.

**Pull request merged**